File: generated-sources/javascript-closure-angular/mojang-sessions/API/Client/PlayerSkinMetadata.js
goog.provide('API.Client.PlayerSkinMetadata');

/**
 * Contains details about the skin
 * @record
 */
API.Client.PlayerSkinMetadata = function() {}

/**
 * The player model, currently only \"slim\" (Alex) is valid, for Steve's model this property must be absent.
 * @type {!string}
 * @export
 */
API.Client.PlayerSkinMetadata.prototype.model;

/** @enum {string} */
API.Client.PlayerSkinMetadata.ModelEnum = { 
  slim: 'slim',
}

File: generated-sources/scala-gatling/mojang-sessions/src/gatling/scala/org/openapitools/client/model/PlayerSkinMetadata.scala

package org.openapitools.client.model


case class PlayerSkinMetadata (
    /* The player model, currently only \"slim\" (Alex) is valid, for Steve's model this property must be absent. */
    _model: Option[String]
)
object PlayerSkinMetadata {
    def toStringBody(var_model: Object) =
        s"""
        | {
        | "model":$var_model
        | }
        """.stripMargin
}
